<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1404</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1191</width>
      <height>741</height>
     </rect>
    </property>
    <property name="viewMode">
     <enum>QMdiArea::TabbedView</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>40</y>
      <width>131</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>按钮1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>按钮2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>单选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1404</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMdi">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionTab"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_9"/>
   </widget>
   <addaction name="menuMdi"/>
   <addaction name="menu"/>
   <addaction name="menuView"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>新建文件</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionTab">
   <property name="text">
    <string>Tab模式</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>级联</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>平铺</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>平铺</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
